geode-core/src/main/java/org/apache/geode/internal/cache/EntryEventImpl.java

```diff
@@ -1719,7 +1716,7 @@ private void setNewValueInRegion(final InternalRegion owner, final RegionEntry r
       int vSize;
       Object ov = basicGetOldValue();
       if (ov instanceof CachedDeserializable && !(GemFireCacheImpl.DELTAS_RECALCULATE_SIZE
-          || this.forceRecalculateSize)) {
+          || ((org.apache.geode.Delta) v).getForceRecalculateSize())) {
```

Comment:
Would it be possible to import Delta so we these instanceof and casts get a bit shorter?

Reply:
Done!

Reply:
Done!